src/platform/src/Bridge/Voyage/ModelClient.php

```diff
@@ -34,11 +34,16 @@ public function supports(Model $model): bool
 
     public function request(Model $model, object|string|array $payload, array $options = []): RawHttpResult
     {
+        $modelOptions = $model->getOptions();
```

Comment:
Please use the `$options` parameter of the method instead, it is a result of the model's options and the `invoke` method and allows users to overwrite once defined options from request to request, see `Platform::invoke`:

https://github.com/symfony/ai/blob/e7bec238e4f695da16ce00843ad0d7d7100d27b7/src/platform/src/Platform.php#L47-L50